resources/settings.xml

```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <settings>
-    <setting id="last_update" value="" visible="false" />
-    <setting id="version" value="" visible="false" />
+    <setting id="last_update" value="" visible="false"/>
+    <setting id="ish_version" value="" visible="false"/>
```

Comment:
We need this in place as "version" as this is a standardized way to find if the addon has been upgraded and take action.

Reply:
Alright, I just thought since we have chromeos version now, too, it would be easier to distinguish.